client-sdk/ts-web/core/package.json

```diff
@@ -13,13 +13,12 @@
     },
     "dependencies": {
         "bech32": "^1.1.4",
-        "cborg": "^1.1.2",
-        "elliptic": "^6.5.3",
+        "cborg": "^1.0.4",
```

Comment:
Why the downgrade?

Reply:
Ah, that's a typo from switching between branches, my bad. Will fix that right now.